### DISCUSSION TITLE
When changing camera angle in sequencer, sequencer doesn't create keys

### DISCUSSION THREAD

**Discussion by onyxlee** [microdee/OpenUnrealSpaceMouse#100] [Q&A]
Hi guys, we all know, in the UE sequencer, when you have that auto key button pressed, when you use mouse to adjust the camera angle or anything, sequencer will automatically create a key to store that change. But when I use space mouse and its UE5.3 plugin, the key will not be stored. Any changes you make on the camera will be lost whenever you scrub the timeline.
Is there any secret setting or toggle to solve this issue? Thanks a lot.

**Comment by microdee** [comment]
and just to be clear, you're explicitly piloting the correct camera, right? Otherwise unfortunately I haven't touched Unreal sequencer in two years now, so I have no idea if they changed something, or if they did, how to fix that :(

**Comment by onyxlee** [comment]
Thank you, David. Yeah, I was piloting the correct camera using space
mouse, it doesn't create a key. When I manually press the button to create
a key, it's a wrong one. The camera will face the wrong direction. But if I
pilot using my mouse, it's all good. Strange, right?

**Reply by microdee** [reply]
Yeah that is indeed painful, damn, well unfortunately I don't have the resources right now to look into this deeper. If you know your Unreal C++ you might be able to also look into this. This is where I tell the viewport to update necessary camera positions: https://github.com/microdee/UE4-SpaceMouse/blob/c05da0361f80f736236ef8a0baf5033db2e2a42a/Source/SpaceMouseEditor/Private/SmEditorManager.cpp#L416 . It did rely on internal processes of the viewport to take effect, so I can completely imagine they change that.
